cmd/releasego/build-pipeline.go

```diff
@@ -88,71 +110,161 @@ func handleBuildPipeline(p subcmd.ParseFunc) error {
 
 	// Make our own client. The AzDO library doesn't support the 7.1 API needed to pass parameters:
 	// https://docs.microsoft.com/en-us/rest/api/azure/devops/build/builds/queue?view=azure-devops-rest-7.1
-	client := new(http.Client)
+	var client http.Client
```

Comment:
try not to use default client, they are lacking of timeouts.

Reply:
Good call. We haven't seen a timeout happen here, but there's a first time for everything, and it would hold up the release if it does.